integration_test/ops_agent_test.go

```diff
@@ -2091,6 +2093,386 @@ func TestPrometheusMetricsWithJSONExporter(t *testing.T) {
 	})
 }
 
+// Test the Histogram and Summary metric types using static testing files
+// The files will contain metrics in the right format and
+// hosted by a simple Python HTTP server so that the agent can scrape the metrics
+// The test will send two sets of metric points, to verify the cumulative metrics
+// are correctly received and processed
+func TestPrometheusMetricsHistogramAndSummary(t *testing.T) {
```

Comment:
[super ultra nit]: This is a pretty massive test. I wonder if we can simplify some of this into a separate method for the set up and then have two tests: 1 for histogram and 1 for summary. That way if one of them breaks, debugging and fixing may be easier

Reply:
Good suggestion! I have broken this into two separate tests and look cleaner this way.